main.py

```diff
@@ -1,55 +1,205 @@
+import datetime
+from enum import Enum
+
+import matplotlib.pyplot as plt
+import numpy as np
 import pandas as pd
+import seaborn
+from sklearn import metrics
+from sklearn.linear_model import LinearRegression
+from sklearn.model_selection import train_test_split
+from sklearn.svm import LinearSVR
+from sklearn.pipeline import make_pipeline
+from sklearn.preprocessing import StandardScaler
+
+
+class Function(Enum):
+    INFO = 1
+    TEMP_MAX_HISTPLOT = 2
+    TEMP_MAX_FACEGRID = 3
+    PRECIP_HISTPLOT = 4
+    WEATHER_COUNTPLOT = 5
+    WEATHER_PIECHART = 6
+    LR_DEFAULT = 7
+    LR_RANDOM = 8
+    SVR_DEFAULT = 9
+
+
+def dataframe_add_months_years(dataframe: pd.DataFrame) -> pd.DataFrame:
+    months = []
+    years = []
+    for date in dataframe['date']:
+        months.append(int(date.month))
+        years.append(int(date.year))
```

Comment:
There surely is a one-liner to extract the `month` and `day` as columns in a dataframe if we convert the `date` from `object` to `datetime`.